src/utilities/spacing/spacing.scss

```diff
@@ -17,11 +17,17 @@
     }
   }
 
+  /**
+    * 1. Ideally we could use the margin-inline shorthand, but it is not as well
```

Comment:
This is `block`, not `inline`. Should probably just be `*-block` shorthand (since this covers `margin` _and_ `padding`).

Reply:
Sorry, copy-paste error. Fixing now.

Reply:
Both updated.